### PR TITLE
DM-16236 -- Make it possible to run corner chips through ISR

### PR DESCRIPTION
**Pull request by SimonKrughoff**

### CODE REVIEW

File: python/lsst/obs/lsst/phosim.py [hunk 1]
```diff
@@ -73,11 +73,7 @@ def translate_filter(self, md):
         filter : `str`
             filter name
         """
-        raftName = self.translate_raftName(md)
-        if raftName in ["R00", "R04", "R40", "R44"]:
-            return "NONE"
-        else:
-            return md.get("FILTER")
+        return md.get("FILTER")
```
Comment (mfisherlevine) [inline python/lsst/obs/lsst/phosim.py]: The code change itself seems fine, but I'm kind of curious about why these were deliberately and specifically excluded before. Do you have any idea why that is, and hence why this change is necessary? I just worry that it it's far from an accident that's being reversed here.

File: policy/lsstCamMapper.yaml [hunk 2]
```diff
@@ -203,6 +203,18 @@ datasets:
     python: lsst.pipe.tasks.indexReferenceTask.IngestIndexedReferenceConfig
     storage: ConfigStorage
     template: config/IngestIndexedReferenceTask.py
+  runIsr_config:
```
Comment (mfisherlevine) [inline policy/lsstCamMapper.yaml]: Perhaps none of these were in `obs_base` when you made these additions, but they seem to be now, so I think this can probably be cut down to only specifying the template for the `runIsr_metadata`, the rest might just work now? Otherwise, see the individual comments on the rest of this block.

File: policy/lsstCamMapper.yaml [hunk 3]
```diff
@@ -203,6 +203,18 @@ datasets:
...
+    persistable: Config
+    python: lsst.meas.mosaic.isrTask.IsrTaskConfig
```
Comment (mfisherlevine) [inline policy/lsstCamMapper.yaml]: Why does this differ from the nominal `lsst.ip.isr.RunIsrConfig` specified in `obs_base`?

File: policy/lsstCamMapper.yaml [hunk 4]
```diff
@@ -203,6 +203,18 @@ datasets:
...
+    storage: ConfigStorage
+    tables: raw
```
Comment (mfisherlevine) [inline policy/lsstCamMapper.yaml]: Does this need a table? One isn't specified in `obs_base`...

File: policy/lsstCamMapper.yaml [hunk 5]
```diff
@@ -203,6 +203,18 @@ datasets:
...
+    template: config/isr.py
```
Comment (mfisherlevine) [inline policy/lsstCamMapper.yaml]: Shouldn't this by `runIsr.py`?

File: policy/lsstCamMapper.yaml [hunk 6]
```diff
@@ -203,6 +203,18 @@ datasets:
...
+  runIsr_metadata:
+    persistable: PropertySet
+    python: lsst.daf.base.PropertySet
+    storage: YamlStorage
+    tables: raw
+    template: metadata/Isr.yaml
```
Comment (mfisherlevine) [inline policy/lsstCamMapper.yaml]: Should this be a capital `I` in `Isr.yaml`?

File: policy/lsstCamMapper.yaml (hunk 2 above)
Comment (SimonKrughoff) [reply policy/lsstCamMapper.yaml]: OK.  This is right.  I've reverted the change, so now it only sets the metadata.  Hopefully this helps.
